### DISCUSSION TITLE
How to set follow_symlink to True?

### DISCUSSION THREAD

**Discussion by kyuhyong** [zauberzeug/nicegui#2720] [Q&A] [answered]
### Question

In https://nicegui.io/documentation/section_pages_routing#add_a_directory_of_static_files_
There is not much explanation about how to change follow_symlink to True. 
How I can set follow_symlink = True ?

**Comment by falkoschindler** [comment, marked as answer]
Have you tried
```py
app.add_static_files('/examples', 'examples', follow_symlink=True)
```

**Reply by kyuhyong** [reply]
Last time I tried the code gave me "Too many argument error" and now it is ok.
Don't know what I did wrong but it is working now. :)

**Reply by kyuhyong** [reply]
I think before entering this command, I got error 'add_static_files() got an unexpected keyword argument 'follow_symlink''
```
pip install --upgrade nicegui
```
Above upgrade solved the problem actually.
